### PR TITLE
[LCS-407] Fix- Subject email should say 1 tenant (singular)

### PR DESCRIPTION
**Pull request by sulthan-ahmed**

### CODE REVIEW

File: apps/right-to-rent-check/behaviours/caseworker-email.js [hunk 1]
```diff
@@ -5,8 +5,12 @@ const path = require('path');
 
 const getSubjectWithTotalTenants = (model, translate) => {
   const totalTenants = model.tenants.length;
-  return translate('email.caseworker.subject.prefix') +
-    totalTenants + translate('email.caseworker.subject.suffix');
+  let subject = translate('email.caseworker.subject.prefix') +
+    totalTenants;
```
Comment (JoeChapman) [inline apps/right-to-rent-check/behaviours/caseworker-email.js]: Why are these on two lines, IMO it's easier to read on one line.

File: apps/right-to-rent-check/behaviours/caseworker-email.js [hunk 2]
```diff
@@ -5,8 +5,12 @@ const path = require('path');
 
 const getSubjectWithTotalTenants = (model, translate) => {
   const totalTenants = model.tenants.length;
```
Comment (JoeChapman) [inline apps/right-to-rent-check/behaviours/caseworker-email.js]: I know this isn't your code, but this should be using a getter `model.get('tenants').length`

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/behaviours/caseworker-email.js]: actually it was me originally, why does it make a difference?

File: apps/right-to-rent-check/behaviours/caseworker-email.js (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/right-to-rent-check/behaviours/caseworker-email.js]: you're right it should be on one line, I'll amend

File: apps/right-to-rent-check/behaviours/caseworker-email.js (hunk 2 above)
Comment (sulthan-ahmed) [reply apps/right-to-rent-check/behaviours/caseworker-email.js]: I tried `model.get('tenants').length` but I get `model.get is not a function`, is this model the same as a hof-model?

**Pull request merged**